pkg/registry/core/next/nsm_registry_server.go

```diff
@@ -63,14 +63,9 @@ func (n *nextNSMRegistryServer) GetEndpoints(ctx context.Context, in *empty.Empt
 func NewWrappedNSMRegistryServer(wrapper NSMRegistryServerWrapper, clients ...registry.NsmRegistryServer) registry.NsmRegistryServer {
 	rv := &nextNSMRegistryServer{servers: make([]registry.NsmRegistryServer, 0, len(clients))}
 	for _, c := range clients {
-		rv.servers = append(rv.servers, wrapper(c))
+		if c != nil {
+			rv.servers = append(rv.servers, wrapper(c))
+		}
 	}
 	return rv
 }
-
-// NewNSMRegistryServer - chains together clients into a single registry.NSMRegistryServer
-func NewNSMRegistryServer(servers ...registry.NsmRegistryServer) registry.NsmRegistryServer {
```

Comment:
Not sure that we should delete this.

Reply:
I is moved, not deleted.

Reply:
pkg/registry/core/chain/nsm_registry_server.go:26